vagga.yaml

```diff
@@ -100,12 +111,12 @@ commands:
       app: *run-app
       db: *run-db
 
-  test: !Command
-    description: Run tests
-    container: test
+  lint: !Command
+    description: Run linters
+    container: lint
     run: pre-commit run --all-files
 
-  pre-commit: !Command
-    description: Run pre-commit
+  test: !Command
+    description: Run tests
     container: test
-    run: [pre-commit]
+    run: python3 manage.py test --pattern="*_test.py"
```

Comment:
Yes, it didn't recognize any. I'm not sure, but I guess, that in docs was mentioned that test files should be named as 'test_*.py' to be recognized by this test-runner. But pre-commit hook doesn't allow this name pattern.